src/components/textarea/__snapshots__/textarea.test.jsx.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Textarea has defaults 1`] = `
+<body>
+  <div>
+    <div
+      role="presentation"
+    >
+      <label
+        class="label"
+        for="foo"
```

Comment:
AFAIK, `for` works for `<input>`s. I wonder if it works for `<textarea>`s. We might need to use `aria-labelledby` instead -- and I think that's part of the `FormControl` API already.

Reply:
It seems to work, but hopefully QA can be more thorough than I am. 😅